lib/collector/parse-response.js

```diff
@@ -55,6 +55,7 @@ module.exports = function parse(name, response, callback) {
         logger.trace(json, 'Deserialized from collector:')
 
         payload = json.return_value || payload
+        // This is ignoring any `validations` present on the returned JSON.
```

Comment:
I'm not sure if we want to be doing this? I just noted it because I encountered it while verifying that I migrated a test successfully. If we want to solve it, that should be a different issue.

Reply:
I have no idea what this is even talking about